index.js

```diff
@@ -151,11 +153,17 @@ app.get('/snippet.mp3', (req, res) => {
     text = formatContentForReading.wrapAndProcessItemData(itemData);
   }
 
-  autovoice.snippetMp3(text, voice)
-  .then(mp3Content => {
-    res.set('Content-Type', 'audio/mpeg');
-    res.send(mp3Content);
-  })
+  if (text.length > AWS_POLLY_CHAR_MAX) {
```

Comment:
Wouldn't the solution here be splitting the string rather than throw an error?

Reply:
This is only for the demo route, and gives immediate feedback that the sample text is too long. If it silently dropped the excess text and TTSed the initial part, you might waste time editing the text that is not reaching TTS system.